driver/driver_options_test.go

```diff
@@ -73,6 +74,29 @@ func TestDriverOptions_ConnectTimeout(t *testing.T) {
 	conn.Close()
 }
 
+func TestDriverOptions_BufferSize(t *testing.T) {
+	log.SetLevel(log.LevelDebug)
+	srv := CreateMockServer(t)
+	defer srv.Stop()
+
+	SetDSNOptions(map[string]DriverOption{
```

Comment:
our application is already using `SetDSNOptions` so we're planning on using the same technique for the `BufferSize`. I didn't want to add more configuration to the public README of this project because I felt that the default for most people is probably fine and therefore no need to add more configuration options.